app/workers/aggregations_dump_worker.rb

```diff
@@ -1,29 +1,9 @@
 class AggregationsDumpWorker
   include Sidekiq::Worker
-  include DumpWorker
-  include RateLimitDumpWorker
 
   sidekiq_options queue: :data_high
 
-  def perform_dump
-    raise ApiErrors::FeatureDisabled unless Panoptes.flipper[:dump_worker_exports].enabled?
-    # Set expiry time of signed_url to one day from now
-    medium.update!(put_expires: 1.day.from_now.to_i - Time.now.to_i)
-
-    AggregationClient.new.aggregate(resource, medium)
```

Comment:
Maybe we should look at purging the aggregation client if we aren't using it? can come later but best not to leave it laying around.

Reply:
Yeah I want to make a second PR removing AggregationClient and the API to trigger the worker.

Reply:
👍